### PR TITLE
Switch chardet for Charset Normalizer

### PR DESCRIPTION
**Pull request by BLKSerene**
`Requests` recently has switched `chardet` which is unmaintained for a long time for a better alternative `Charset Normalizer`.

As `chardet` is only used in `underthesea/corpus/validate_corpus.py`, this pull requests did the same thing, and `chardet` could then be removed from the dependency list. (Charset Normalizer is already a dependency of `requests` which `underthesea` is depended on.)

### CODE REVIEW

**Comment by rain1024** [comment]
@BLKSerene Looks good to me. Please rebase master to trigger github actions for CI

**Comment by BLKSerene** [comment]
Done, please approve running CI

File: underthesea/corpus/validate_corpus.py [hunk 1]
```diff
@@ -57,16 +57,15 @@ def validate_corpus_type(corpus_type):
...
-            if detector.done or i > 1000:
+            if i < 1000:
+                text += line
+            else:
                 break
-    detector.close()
-    result = detector.result
-    if not (result["encoding"] == "utf-8" and result["confidence"] >= 0.99):
+    results = charset_normalizer.from_bytes(text).best()
+    if not (results.encoding == "utf-8" and result.coherence >= 0.99):
```
Comment (rain1024) [inline underthesea/corpus/validate_corpus.py:68]: It should be `results.coherence`

**Review by rain1024 — COMMENTED** [review]
please fix this little typo

File: underthesea/corpus/validate_corpus.py (hunk 1 above)
Comment (BLKSerene) [reply underthesea/corpus/validate_corpus.py]: Fixed

**Pull request merged**